build/docker/deploySchemaAndTenant.sh

```diff
@@ -25,7 +25,7 @@ do
   EXIT_CODE="${PIPESTATUS[0]}"
   LOG_OUT=`cat ${TMP_FILE}`
   if [ "$EXIT_CODE" == "0" ]
```

Comment:
better to use 
```
if [[ ... ]]; then
    ...
```
the double bracket is a built-in test command.

Reply:
```suggestion
  if [[ "$EXIT_CODE" == "0" ]]; then
```